launch/tier4_autoware_api_launch/launch/include/external_api_adaptor.launch.py

```diff
@@ -41,11 +45,15 @@ def generate_launch_description():
         _create_api_node("metadata_packages", "MetadataPackages"),
         _create_api_node("route", "Route"),
         _create_api_node("service", "Service"),
-        _create_api_node("start", "Start"),
         _create_api_node("vehicle_status", "VehicleStatus"),
         _create_api_node("velocity", "Velocity"),
         _create_api_node("version", "Version"),
     ]
+    if LaunchConfiguration("launch_calibration_status_api").perform(context) == "true":
```

Comment:
I think it's better to pass the argument `condition=IfCondition(LaunchConfiguration("arg name"))` to `_create_api_node` than `OpaqueFunction`.

Reply:
@isamu-takagi 
Sorry, I don't understand at all. Could you give me more detailed information? (I'm almost new to launch.py)

Reply:
@takayuki5168 
Sorry, it couldn't specify 'condition' argument directly to the ComposableNode. So please do as follows. I think it would be convenient to make this a function.
```
loader = LoadComposableNodes(
    composable_node_descriptions=[_create_api_node("calibration_status", "CalibrationStatus")],
    target_container=container,
    condition=IfCondition(LaunchConfiguration("arg_test")),
)
```

Reply:
Thank you.
https://github.com/autowarefoundation/autoware.universe/pull/1324/commits/47fb3b1dc88b862170852136c575da55ad9dc319
DONE

Reply:
OpaqueFunction can be removed.

Reply:
https://github.com/autowarefoundation/autoware.universe/commit/134136ec154284a912c9ac92fdfd7c72257bb125
https://github.com/autowarefoundation/autoware.universe/commit/c3872d0a285fa92b369c07781cc3f4db2e886e63
DONE